### DISCUSSION TITLE
Contributing.md

### DISCUSSION THREAD

**Discussion by Sh0g0-1758** [rust-bitcoin/rust-bitcoin#2567] [General]
I was going through contributing.md and I saw no information about the pinging policy. So do you guys follow the 1 ping per week policy or is it different here?

**Comment by Sh0g0-1758** [comment]
Also, I think it would be better to better explain the atomic commits, it refers to having only one commit in a PR for better review but new contributors might not be aware of this since in most open source orgs, this policy is not followed.

**Reply by apoelstra** [reply]
There is no open-source org where "have a coherent set of commits" is discouraged. "Not followed", perhaps, in orgs which have more forgiving or patient maintainers. But I think it's common sense that your PRs should be as elegant and coherent as possible, just like your code should be.

**Reply by Sh0g0-1758** [reply]
Hm... True, got your point 👍

**Comment by apoelstra** [comment]
And sure, I think if you feel you've been forgotten about, and you ping once a week, nobody would be too put out by that.

**Reply by Sh0g0-1758** [reply]
Great 👍, just needed a confirmation.
